index.html

```diff
@@ -0,0 +1,27 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <nav class="nav">
+        <a href="">video</a>
+        <a href="">sidebar</a>
+    </nav>
+    
+    
+    
+    <iframe class="video" width="560" height="315" src="https://www.youtube.com/embed/Yg6UhhV_K1s" 
+            title="YouTube video player" 
+            frameborder="0" 
+            allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" 
+            allowfullscreen controls>
+    </iframe>
```

Comment:
Kindly wrap the <iframe> tag in the <main> tag as this helps define the different sections of your page and improves your page's search engine optimization.

Reply:
- [ ]Great job mentioning this to your student in a polite way and adding an explanation. However, the comment is still unclear as the comment above. Which kind of tag should the student use? Giving more context and information can help the student to address the problem.

**SEE AN EXAMPLE BELOW**:

I kindly suggest that to wrap the `<nav>` tag in a `<whatever>` tag as this helps better define the different sections of your page and improves your page's search engine optimization.

[OPTIONAL] There's an unnecessary break of line in the comment. Kindly address this issue so your comment can look more professional.